src/lib.rs

```diff
@@ -203,23 +203,59 @@ pub struct Metrics {
     pub strikeout_thickness: f32,
 }
 
-pub trait Rasterize {
-    /// Errors occurring in Rasterize methods.
-    type Err: ::std::error::Error + Send + Sync + 'static;
+/// Errors occuring when using the rasterizer.
+#[derive(Debug)]
+pub enum Error {
+    /// Unable to find a font matching the description.
+    FontNotFound(FontDesc),
+
+    /// Unable to find metrics for a font face.
+    MetricsNotFont,
```

Comment:
Is this a possible typo of `MetricsNotFound` ?

Reply:
Thx.fixed.